velox/functions/prestosql/tests/ArithmeticTest.cpp

```diff
@@ -821,5 +821,44 @@ TEST_F(ArithmeticTest, wilsonIntervalUpper) {
   EXPECT_DOUBLE_EQ(wilsonIntervalUpper(1, 3, kInf).value(), 1.0);
 }
 
+TEST_F(ArithmeticTest, cosineSimilarity) {
+  const auto testCosineSimilarity =
+      [&](const std::vector<
```

Comment:
Why do we take a list of maps? This is a simple function, hence, no need to test it on multi-row vectors. Let's simplify the test.